### PR TITLE
GDB 8276 Introduce custom fields for the newly added data sources in ontop repository

### PR DESCRIPTION
**Pull request by DenitsaStoianova**
Introduced custom fields for Dremio, Databricks and Snowflake drivers, add them to cypress tests and update JDBC download link to be Google search for “ jdbc download”

### CODE REVIEW

**Comment by DenitsaStoianova** [comment]
Jenkins retry build

File: test-cypress/integration/repository/repositories.spec.js [hunk 1]
```diff
@@ -628,7 +628,64 @@ describe('Repositories', () => {
         testOntopConfigurationElementsVisibility('Lenses file', '#lensesFile');
         testOntopConfigurationElementsVisibility('Ontology file', '#owlFile');
         //verify driver download url
-        compareDriverDownloadUrl('https://www.ibm.com/support/pages/db2-jdbc-driver-versions-and-downloads');
+        compareDriverDownloadUrl('https://www.google.com/search?q=db2+jdbc+download');
+
+        //Select Dremio driver type and verify elements and download url (if available)
+        selectDatabaseDriver('Dremio');
```
Comment (yordanalexandrov) [inline test-cypress/integration/repository/repositories.spec.js:634]: When you add a default to the switch, also change this test, to check if going back to the other drivers would work ok.

**Review by yordanalexandrov — CHANGES_REQUESTED** [review]

File: src/js/angular/repositories/ontop-repo.directive.js [hunk 2]
```diff
@@ -75,12 +75,33 @@ function ontopRepoDirective($uibModal, RepositoriesRestService, toastr, Upload,
...
         };
 
+        $scope.constructCustomFiledForDriver = function (driver) {
+            switch (driver) {
+                case 'snowflake':
+                    $scope.supportedDriverLabels.hostName = 'Account identifier';
+                    $scope.supportedDriverLabels.databaseName = 'Warehouse';
+                    break;
+                case 'databricks':
+                    $scope.supportedDriverLabels.databaseName = 'HttpPath';
```
Comment (yordanalexandrov) [inline src/js/angular/repositories/ontop-repo.directive.js:93]: The hostname for `dratabricks` and `dremio` will also be called `Account identifier` if `snowflake` is selected first.

Is this ok?

Comment (DenitsaStoianova) [reply src/js/angular/repositories/ontop-repo.directive.js:95]: It must be callled 'Hostname' for all other data sources, fixed in commit: 710a16d366ede3992d3931b4a6d3c4bd56b00933

**Review by yordanalexandrov — APPROVED** [review]

**Pull request merged**